### ISSUE TITLE
Sidebar only partially visible in android.

### ISSUE THREAD

**Issue by not-much-io** [luminus-framework/luminus#74]
When opening http://www.luminusweb.net/docs in android google chrome the sidebar is only partially visible. I am not zoomed in. 
![screenshot_2015-06-02-16-46-19](https://cloud.githubusercontent.com/assets/5730578/7937303/fe9fd996-0947-11e5-96a5-4ced217e8333.png)

**Comment by yogthos** [comment]
a workaround at the moment is to turn to landscape and back

**Comment by not-much-io** [comment]
Yes, that did work. So I take it this is a known bug?

**Comment by yogthos** [comment]
yup only seems to affect chrome on android and for small screen size, I haven't found why it doesn't figure out the screen size correctly
